rllib/models/torch/complex_input_net.py

```diff
@@ -115,6 +115,7 @@ def __init__(self, obs_space, action_space, num_outputs, model_config, name):
                     name="one_hot_{}".format(i),
                 )
                 concat_size += self.one_hot[i].num_outputs
+                self.add_module("discrete_{}".format(i), self.cnns[i])
```

Comment:
a stupid question, where are these names defined? discrete_() and flat_{} I mean.
we seem to be making an assumption of how get_model_v2 would name the models created.

Reply:
@gjoliver it shouldn't matter what it is named, this is necessary to register the model variables with the torch module

Reply:
This PR needs to get updated before merging though, not use self.cnns